Automattic/wp-calypso
# Github Deployments: Add Github Deployments Survey

Closes https://github.com/Automattic/dotcom-forge/issues/6120

## Proposed Changes
This PR adds Github Deployment survey 

* Add `overlay` option to the modal so we can hide the overlay
* Hide survey for 7 days if the user has deployments or creates on for the first time.

## Testing Instructions



* go to `/github-deployments/siteSlug`
* If you already have deployments you should see two cookies in cooke explorer `github-deployments-has-deployments` and `github-deployments-hide-survey`. If you create for the first time they should also appear
* delete `github-deployments-hide-survey` and refresh.
* Verify the cookie modal pops up and you are redirected to the survey (**Please do not answer the survey**)

**Survey is only show to english locale**


![image](https://github.com/Automattic/wp-calypso/assets/47489215/2ed99c81-cfdd-4339-930b-cae926ae66bc)


## Pre-merge Checklist
